dot/rpc/modules/author_integration_test.go

```diff
@@ -50,7 +52,14 @@ func TestMain(m *testing.M) {
 }
 
 func TestAuthorModule_Pending(t *testing.T) {
-	txQueue := state.NewTransactionState()
+	ctrl := gomock.NewController(t)
+	telemetryMock := telemetry.NewMockTelemetry(ctrl)
+	telemetryMock.
+		EXPECT().
+		SendMessage(gomock.Any()).
+		AnyTimes()
+
```

Comment:
expect concrete args and times (or expect multiple different things)

Reply:
This file is related to the author integration test PR, once this PR was merged I will address this there!